WPFWijkagent/WijkagentTests/ScraperTest.cs

```diff
@@ -46,13 +46,13 @@ public void GetSocialMediaMessages_offenceGiven_IsInstance()
             //arrange
             Offence offence = new Offence()
             {
-                LocationID = new Location(52.501127, 6.0789937),
-                DateTime = new DateTime(2019, 11, 25)
+                LocationID = new Location(52.37317168, 4.8936582),
+                DateTime = new DateTime(2019, 12, 4)
             };
             Scraper s = new Scraper(offence);
             //act
             List<SocialMediaMessage> socials = s.GetSocialMediaMessages();
-            Console.WriteLine(socials.Count);
+            //Console.WriteLine(socials.Count);
```

Comment:
Ik zou deze hele lijn eruit halen. Commented code hoort niet thuis in de master.

Reply:
aangepast

Reply:
Top